react/week4/App.js

```diff
@@ -0,0 +1,48 @@
+import React, { Fragment, Component } from "react";
+
+import "./App.css";
+import Banner from "./banner/banner.component";
+import SearchBox from "./search-box/search-box.component";
+import Card from "./card/card.component";
+
+class App extends Component {
```

Comment:
we dont use class based approch any more I am not sure why we have class here , there should be component.